api/routes/sample.py

```diff
@@ -233,3 +233,15 @@ async def get_history_of_sample(
     result = await st.get_history_of_sample(internal_id)
 
     return result
+
+
+@router.post('/created', operation_id='getSampleCreateDate')
```

Comment:
```suggestion
@router.post('/created', operation_id='getSamplesCreateDate')
```

Reply:
Could we expand the url path to less likely to clash later?

Reply:
Sure